packages/cli/lib/functions.js

```diff
@@ -0,0 +1,18 @@
+const moment = require('moment');
+
+const getFunctionArrays = resp => {
+  return resp.objects.map(func => {
+    const { route, method, created, updated, secretNames } = func;
+    return [
+      route,
+      method,
+      secretNames.join(', '),
+      moment(created).format(),
+      moment(updated).format(),
+    ];
+  });
+};
+
+module.exports = {
+  getFunctionArrays,
+};
```

Comment:
Might make sense to just rename this file `getFunctionArrays` since `functions` isn't very descriptive